sources/parse-datetime/src/lib.rs

```diff
@@ -58,23 +61,26 @@ pub fn parse_datetime(input: &str) -> Result<DateTime<Utc>> {
     // Otherwise, pull apart a request like "in 5 days" to get an exact datetime.
     let mut parts: Vec<&str> = input.split_whitespace().collect();
     ensure!(
-        parts.len() == 3,
+        parts.len() == 3 || parts.len() == 2,
         error::DateArgInvalid {
             input,
-            msg: "expected RFC 3339, or something like 'in 7 days'"
+            msg: "expected RFC 3339, or something like 'in 7 days' or '7 days'"
         }
     );
     let unit_str = parts.pop().unwrap();
     let count_str = parts.pop().unwrap();
-    let prefix_str = parts.pop().unwrap();
 
-    ensure!(
-        prefix_str == "in",
-        error::DateArgInvalid {
-            input,
-            msg: "expected RFC 3339, or prefix 'in', something like 'in 7 days'",
-        }
-    );
+    // the prefix string 'in' is optional
+    let maybe_prefix_str = parts.pop();
+    if let Some(prefix_str) = maybe_prefix_str {
```

Comment:
```suggestion
    if let Some(prefix_str) = parts.pop() {
```
Makes it clearer we're not using it outside.